packages/ckeditor5-html-support/docs/features/general-html-support.md

```diff
@@ -233,6 +235,49 @@ htmlSupport: {
 
 The above configuration will work similarly to [`allowedContent: true`](/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-allowedContent) option from CKEditor 4.
 
+## HTML comments
+
+By default, all HTML comments are filtered out during the editor initialization. The {@link module:html-support/htmlcomment~HtmlComment} feature allows developers to keep comments in the document content and retrieve them back, i.e. during {@link builds/guides/integration/saving-data saving editor data}. Comments are transparent from the user point of view and they are not displayed in the editable element.
+
+Please note that the support for HTML comments is at the basic level so far - see the [known issues](#known-issues) section below.
+
+<info-box>
+	The HTML comment feature is **experimental and not yet production-ready**.
+</info-box>
+
+### Demo
+
+The CKEditor 5 instance below is configured to keep the HTML comments in the document content. You can view the source of the document using {@link features/source-editing source editing} feature. Toggle the source editing mode {@icon @ckeditor/ckeditor5-source-editing/theme/icons/source-editing.svg Source editing} to see that HTML comments are present in the document source. You can uncomment the paragraph that is below the picture and after leaving the source editing mode, you will see this paragraph in the editable area.
+
+{@snippet features/html-comment}
+
+### Installation
+
+To add this feature to your rich-text editor, install the [`@ckeditor/ckeditor5-html-support`](https://www.npmjs.com/package/@ckeditor/ckeditor5-html-support) package:
+
+```plaintext
+npm install --save @ckeditor/ckeditor5-html-support
+```
+
+Then add it to the editor configuration:
+
+```js
+import HtmlComment from '@ckeditor/ckeditor5-html-support/src/htmlcomment';
+
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ HtmlComment, ... ],
+	} )
+	.then( ... )
+	.catch( ... );
+```
+
+<info-box info>
+	Read more about {@link builds/guides/integration/installing-plugins installing plugins}.
+</info-box>
+
+HTML comment feature does not require any configuration.
+
 ## Known issues
```

Comment:
Maybe we could have one section for known issues in general HTML support and one for HTML comments?